scripts/ci_setup.py

```diff
@@ -161,6 +161,15 @@ def add_arguments(parser: ArgumentParser) -> ArgumentParser:
         help='Directory to install dotnet to'
     )
 
+    parser.add_argument(
+        '--not-in-lab',
+        dest='in_lab',
+        required=False,
+        action='store_false',
+        default=True,
```

Comment:
is my understanding correct that if we set the default value to `True`, the current default behavior is going to change? and we will need to update some CI configuration somewhere?

Reply:
Sorry, this is a bit confusing. I'm using a negated backing variable here `in_lab`, so the default is `in_lab=true`, which IIUC, is the default behavior. And passing `--not-in-lab` would set `store_false`, and invert the behavior.

I was trying to avoid a `if not not_in_lab` condition, but the way I ended up using this, it's not a concern: `'1' if args.in_lab else '0'`

So, I propose that I change this to be `not_in_lab`, and default that to false. Would that be agreeable?